src/marqo/core/structured_vespa_index/structured_vespa_index.py

```diff
@@ -409,7 +409,7 @@ def get_vector_count_query(self):
             'yql': f'select {common.FIELD_VECTOR_COUNT} from {self._marqo_index.schema_name} '
                    f'where true limit 0 | all(group(1) each(output(sum({common.FIELD_VECTOR_COUNT}))))',
             'model_restrict': self._marqo_index.schema_name,
-            'timeout': '5s'
+            'timeout': 5000
```

Comment:
This was an incorrect use of the query method. I am surprised that Vespa didn't error out for `"timeout": "5sms"`